packages/gafl-webapp-service/src/processors/__tests__/licence-type-display.spec.js

```diff
@@ -0,0 +1,12 @@
+import { getPronoun } from '../licence-type-display'
+
+describe('licence-type-display', () => {
+  describe('getPronoun', () => {
+    it('should return your and you, if isLicenceForYou is true', () => {
+      expect(getPronoun(true)).toStrictEqual({ possessive: 'your', personal: 'you' })
+    })
+    it('should return their and they, if isLicenceForYou is false', () => {
+      expect(getPronoun(false)).toStrictEqual({ possessive: 'their', personal: 'they' })
+    })
```

Comment:
Another one where I'm not sure these pronouns are used anymore, so possibly can be removed?